src/app/modules/group/components/group-manager-list/group-manager-list.component.html

```diff
@@ -87,7 +87,16 @@
               <i *ngIf="rowData.canWatchMembers" class="fa fa-check"></i>
             </span>
```

Comment:
When no value on the form has changed, I think the submit button should be disabled

![image](https://user-images.githubusercontent.com/16257159/132699986-92af781c-b098-4ac5-be78-982f87b6226d.png)

Reply:
Fixed. Not sure if there need to do deep checking of object

Reply:
I was hoping we could take advantage of a [`FormGroup.dirty`](https://angular.io/api/forms/AbstractControl#dirty) indicator, but the (boolean|progress)-section component aren't adapted to be used with forms, that's a shame. And adapting those components will take a lot of time for the exact same result :disappointed: 

Let's keep it this way :heavy_check_mark:

Reply:
Yes, exactly, I also supposed to fix it with form module.